test/integration/finance/charging_invoice_states_test.rb

```diff
@@ -47,8 +47,8 @@ class Finance::ChargingInvoiceStatesTest < ActionDispatch::IntegrationTest
 
       @invoice.buyer_account.bought_plan.update_attributes!(name: 'Paid', cost_per_month: 100)
 
-      ThreeScale::Analytics.expects(:track).with(instance_of(User), 'Charged Invoice',
-                                                 {:plan => 'Paid', :period => 'January  1, 1984 - January 31, 1984', :revenue => 100.0})
+      ThreeScale::Analytics.expects(:track).with(@provider.first_admin, 'Charged Invoice',
```

Comment:
```suggestion
      ThreeScale::Analytics.expects(:track).with(@provider.first_admin!, 'Charged Invoice',
```

Simply to get a most understandable error if there isn't any because we expect to have it and not to be nil and track nil :smile: 

https://github.com/3scale/porta/blob/e2dbb0cf87ed7a340261f681b9444254dd1a199b/app/models/account.rb#L193-L199